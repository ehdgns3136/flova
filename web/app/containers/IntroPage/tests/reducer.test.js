
import { fromJS } from 'immutable';
import introPageReducer from '../reducer';

import {
  loginSuccessAction,
} from '../actions';

describe('introPageReducer', () => {
  it('returns the initial state', () => {
    expect(introPageReducer(undefined, {})).toMatchSnapshot();
  });

  it('handles loginSuccessAction', () => {
    expect(introPageReducer(undefined, loginSuccessAction())).toMatchSnapshot();
  });
});
