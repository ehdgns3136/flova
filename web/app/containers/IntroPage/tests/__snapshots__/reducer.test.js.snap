exports[`introPageReducer handles loginSuccessAction 1`] = `
Object {
  "isLoading": false,
  "loginError": false,
}
`;

exports[`introPageReducer returns the initial state 1`] = `
Object {
  "isLoading": false,
  "loginError": false,
}
`;
